### ISSUE TITLE
Setting showHint on uibTypeAhead makes it unable to tab to it

### ISSUE THREAD

**Issue by anzeo** [angular-ui/bootstrap#5478]
Setting the `typeahead-show-hint` option on an input with uib-typeahead makes it unable to use the TAB key to activate the typeahead input.

[Plunkr](http://plnkr.co/edit/gmSb6uvlE0xUWbjWybdW?p=preview)

**Comment by Tom-V** [comment]
true, the `tabindex `of the `hintInputElement `should be set to `-1`.
At the moment you need to tab twice to get to the right element.

**Comment by wesleycho** [comment]
Feel free to open a PR for this - should be an easy fix.

**Comment by anzeo** [comment]
Thanks @deeg!

**Comment by franklindner** [comment]
This still seems to be broken. If I add the show-hint attribute it will always focus the outer element first and on the next tab select the inner input. Everything works as expected if I remove the attribute.
